Dockerfile

```diff
@@ -55,7 +58,10 @@ COPY --from=builder /fossology/dependencies-for-runtime /fossology
 
 WORKDIR /fossology
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+# Fix for Postgres and other packages in slim variant
+RUN mkdir /usr/share/man/man1 \
+ && mkdir /usr/share/man/man7 \
```

Comment:
I prefer the one-liner: `mkdir /usr/share/man/man1 /usr/share/man/man7`

Reply:
Done